esmvaltool/utils/recipe_test_workflow/recipe_test_workflow/app/install_env_file/rose-app.conf

```diff
@@ -1,5 +1,9 @@
+# Copy the site specific environment file to the 'share/bin' directory in the
+# installed Cylc workflow (this directory is automatically added to the
+# ${PATH} by Cylc).
```

Comment:
Would it be possible to adjust this comment to refer to the variables used in the `cp` command below, please? 😊

Reply:
Although, I wonder whether the names of the variables used in the `cp` command below should be updated to be more descriptive (I feel like the names were fine in the Bash script, when it could be seen what the variables were assigned to, but now it's not so clear?).

Reply:
I've made some changes to these variable names in 55bbf3050 and im hoping this means changes to the comment aren't necessary anymore

Reply:
I'm wondering whether `ENV_FILE_SITE_PATH` would be more descriptive than `RTW_ENV_FILE_SOURCE_PATH` (since this is the variable that specifies the site-specific environment file)? Similarly, perhaps `SHARE_BIN_DIR` would be more descriptive than `RTW_ENV_FILE_SHARE_DIR` (since this is the variable that specifies the `share/bin` directory)? 🤔

Reply:
Your're right that makes more sense, i've amended these in commit 7d1c53559 :slightly_smiling_face: